qutebrowser/browser/qtnetworkdownloads.py

```diff
@@ -172,9 +172,9 @@ def retry(self):
         assert not self.successful
         new_reply = self._retry_info.manager.get(self._retry_info.request)
         new_download = self._manager.fetch(new_reply,
+                                           target=self.target,
                                            suggested_filename=self.basename)
         self.adopt_download.emit(new_download)
-        self.cancel()
```

Comment:
In my testing everything seemed fine without the `self.cancel()`. I'll go read through those issues and at tests for them, if they don't exist yet. We'll see what that shakes out :-)